test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```diff
@@ -0,0 +1,66 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.pipeline_alerts.yaml
+
+tests:
+  # ----- Tekton Controller Rapid Restart Tests ----
+  - interval: 1m
+    input_series:
+
+      # see https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/ for explanations of the expanding notation used for the values
+      # tekton pipeline controller is experiencing controller successive restarts so it will be alerted, whereas we are currently not checking
+      # other pods in the same namespace that restart
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-pipelines-controller-a", source_cluster="cluster01"}'
+        values: '1+1x780'
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-pipelines-controller-a", source_cluster="cluster01"}'
+        values: '1+1x780'
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-triggers-controller-a", source_cluster="cluster01"}'
+        values: '2+2x780'
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-triggers-controller-a", source_cluster="cluster01"}'
+        values: '2+2x780'
```

Comment:
Another duplicate time series that should probably be removed.

Reply:
left the time series with the pod label diff and different incrementation to validate this one is not counted  ... but let me know @amisstea if I'm still not entirely in sync with how the unit test time series work

Reply:
This simpler set of time series looks great!